### PR TITLE
Contribution guideline first draft

### PR DESCRIPTION
**Pull request by githubteacher**
@anna, Could you help out and work on this document with me?
- [x] Finish first draft of guiding principles
- [x] Have @anna proof-read
- [ ] At least one colleague :+1: before merging

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # Contribution Guidelines
 
-TBD
+- Topic focus
+- Be nice
+- Be extra nice
+- Be polite
```
Comment (anna) [inline CONTRIBUTING.md]: Should we add "Eat your vegetables"?

**Comment by githubteacher** [comment]
@anna, Could you finish up the first three bullet points in the doc edited in this Pull Request?

**Comment by anna** [comment]
@githubteacher Sorry! I didn't mean to be a slacker. Changes are above.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
 # Contribution Guidelines
 
-TBD
+- Topic focus
+- Be nicer
+- Eat your vegetables
```
Comment (githubteacher) [inline CONTRIBUTING.md]: Perhaps we remove the food theme?

Comment (anna) [inline CONTRIBUTING.md]: Sure! That works for me. Taking care of it now.

**Comment by githubteacher** [comment]
:+1:  @anna, this is good for a first ship.

**Pull request merged**